akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingCustomShardAllocationSpec.scala

```diff
@@ -81,37 +61,23 @@ object ClusterShardingCustomShardAllocationSpec {
 
 }
 
-abstract class ClusterShardingCustomShardAllocationSpecConfig(val mode: String) extends MultiNodeConfig {
+abstract class ClusterShardingCustomShardAllocationSpecConfig(mode: String)
+    extends MultiNodeClusterShardingConfig(
+      mode,
+      additionalConfig = ConfigFactory.parseString(s"""
```

Comment:
I think `additionalConfig` is always a string parsed to config, could perhaps be nudge cleaner if it was accepted as a string and parsed in `MultiNodeClusterShardingConfig` instead?

Reply:
Good, done.